styles.css

```diff
@@ -2,4 +2,41 @@ body {
     background-color: lightseagreen;
 }
 
+.demo{
+    display: flex;
+}
+
+.demo-image img{
+    height: auto;
+    width: 55em;
+    display: inline-block;
+}
+
+.example-circles{
+    background-color: palegoldenrod;
+    height: auto;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.outer-circle{
+    height: 14em;
```

Comment:
This reminds me that we should talk about `em`, `rem`, and `px` next week a bit!